embedded-graphics/src/lib.rs

```diff
@@ -66,12 +74,14 @@
 //!
 //! # Simulator
 //!
-//! Embedded graphics comes with a [simulator]!
+//! Embedded graphics comes with a [simulator]! The simulator can be used to test and debug
+//! embedded graphics code, or produce examples and interactive demos to show of embedded graphics
+//! features.
```

Comment:
Much better than the version in the simulator readme. It sounds like the primary purpose of the simulator is debugging embedded-graphics itself, but I think we should emphasize that it can also be used to do the same for external libraries or applications.

Reply:
I've backported this text to the simulator as well :+1: